### ISSUE TITLE
Bing Map in Modal

### ISSUE THREAD

**Issue by krooshua** [infusion-code/angular-maps#22]
Hi there!
Did someone try to show Bing map with markers in the modal-window? I see an issue in the console when modal-window is closed.
```
ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'draggable' of null
TypeError: Cannot read property 'draggable' of null
    at r.setOptions (d98b7733.js?bu=rms+answers+Maps+AnonymousBegin*SDKPluginStart*SDKInfoboxOverlay*Infobox*SDKColor*Ti…:1)
    at BingMarker.webpackJsonp.../../../../angular-maps/dist/src/models/bing/bing-marker.js.BingMarker.DeleteMarker (bing-marker.js:126)
    at bing-marker.service.js:137
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at Object.onInvoke (core.es5.js:3890)
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:390)
    at Zone.webpackJsonp.../../../../zone.js/dist/zone.js.Zone.run (zone.js:141)
    at NgZone.webpackJsonp.../../../core/@angular/core.es5.js.NgZone.run (core.es5.js:3821)
    at bing-marker.service.js:136
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at r.setOptions (d98b7733.js?bu=rms+answers+Maps+AnonymousBegin*SDKPluginStart*SDKInfoboxOverlay*Infobox*SDKColor*Ti…:1)
    at BingMarker.webpackJsonp.../../../../angular-maps/dist/src/models/bing/bing-marker.js.BingMarker.DeleteMarker (bing-marker.js:126)
    at bing-marker.service.js:137
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at Object.onInvoke (core.es5.js:3890)
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:390)
    at Zone.webpackJsonp.../../../../zone.js/dist/zone.js.Zone.run (zone.js:141)
    at NgZone.webpackJsonp.../../../core/@angular/core.es5.js.NgZone.run (core.es5.js:3821)
    at bing-marker.service.js:136
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke (zone.js:391)
    at resolvePromise (zone.js:783)
    at zone.js:834
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:424)
    at Object.onInvokeTask (core.es5.js:3881)
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:423)
    at Zone.webpackJsonp.../../../../zone.js/dist/zone.js.Zone.runTask (zone.js:191)
    at drainMicroTaskQueue (zone.js:595)
    at webpackJsonp.../../../../zone.js/dist/zone.js.ZoneTask.invokeTask (zone.js:502)
    at ZoneTask.invoke (zone.js:487)
    at timer (zone.js:1829)
```

**Comment by thor-schueler** [comment]
Hi Sergey,

I think we need a bit more information. Could you let us know what version you are using? Also, we would need to see the code that is producing the behavior to figure out what is going on. It would be best if you could create a plunk that reproduces the problem. You should be easily able to do that by forking on of the existing demo plunks (see the Samples wiki page).

Once you have a plunk, we’ll look at it to see what’s going on. Response might be a little slow as we are somewhat under the weather at the moment with Harvey ….

**Comment by krooshua** [comment]
H1 @thor-schueler!

Sure, here the plunker - https://plnkr.co/edit/lCUUCsRphiMMBi6hve5w?p=preview
I've added wrap to map with *ngIf and toggle button. Same error in console.

Maps version - "angular-maps": "^0.6.3",

**Comment by thor-schueler** [comment]
Thanks @krooshua. I see this issue. We will investigate and probably have this resolved in a few days... Please leave the plunk in place for now.

**Comment by thor-schueler** [comment]
This has been addressed in v0.6.6. @krooshua Could you please confirm...

**Comment by thor-schueler** [comment]
Confirmed that the issue does no longer present in the original plunk supplied. Closing this as resolved....
